doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -60,7 +60,7 @@ The issues in this section are related to :ref:`protocols`.
 Amazon Sidewalk
 ===============
 
```

Comment:
```suggestion

.. rst-class:: v2-7-0

KRKNWK-17860: Fatal DFU error in the Sidewalk application mode
  A fatal error occurs when attempting to perform DFU in the Sidewalk application mode instead of the DFU mode. 
  The error involves an assertion in the QSPI, which is in the deep sleep mode to save energy. 

  **Workaround:** The DFU service must be used only when the device is in the DFU mode. 
```